controllers/policyendpoints_controller_test.go

```diff
@@ -748,7 +748,7 @@ func TestArePoliciesAvailableInLocalCache(t *testing.T) {
 
 		mockClient := mock_client.NewMockClient(ctrl)
 		policyEndpointReconciler, _ := NewPolicyEndpointsReconciler(mockClient, logr.New(&log.NullLogSink{}),
-			false, false, false, false, 300)
+			false, false, false, false, 300, 524288)
```

Comment:
Better to use a constant instead for the default size. Also, the default size above is different to our current [default](https://github.com/aws/aws-network-policy-agent/blob/main/pkg/ebpf/c/v4events.bpf.c#L46) in the bpf prog. I think we can leave this @ 256K.

Reply:
Ok, I see `defaultConntrackCacheTableSize ` defined and set to 256K. We can use that here.